app/controllers/external_users/claims_controller.rb

```diff
@@ -132,18 +135,15 @@ def unarchive
   end
 
   def new
+    @claim = resource_klass.new
+    authorize! :new, @claim
```

Comment:
Previously this action was not authorised which meant that any user logged in would be able to access this page as long as they knew the URL. This ensures the action is correctly authorised and only users with permissions to manage this type of claim can do so.